docs/compliance/fisma.md

```diff
@@ -0,0 +1,11 @@
+---
+description: FISMA compliance guidance for Docker Enterprise Edition
+keywords: standards, compliance, security
+title: FISMA
+---
+
+The Federal Information Security Management Act (FISMA) is Federal law that requires the U.S. Federal government to adhere to stringent information security requirements for any and all information systems that it operates. FISMA is implemented via a combination of publications and guidance documents developed by the National Institute of Standrds and Technology (NIST), mandatory Federal Information Processing Standards (FIPS), memorandums and circulars published by the Office of Management and Budget (OMB), and various other agencies and regulatory authorities that are tasked with coordinating information security-related initiatives.
```

Comment:
- No need to say "any and all" (all is good enough) -- it'sj ust extra words with no real added meaning. 
- I try to stay away from "via" ... it's a Latin abbreviation. Say, "FISMA is implemented through a combination" ....  Actually, how about rewriting that sentence like this: 

> FISMA is implemented by adhering to guidance developed by the [National Institute of Standards and Technology (NIST)](https://www.nist.gov/), including [Federal Information Processing Standards (FIPS)](https://www.nist.gov/itl/itl-publications/federal-information-processing-standards-fips), and memoranda by the [Office of Management and Budget (OMB)](https://www.opm.gov/) and other authorities tasked with coordinating information security-related initiatives.

Reply:
Corrected in follow-on commit